app/controllers/api/v1/projects_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class Api::V1::ProjectsController < ApplicationController
+  include JWTHelper
+
+  skip_before_action :verify_authenticity_token
+  before_action :authenticate_jwt
+
+  # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+  def create
+    populate_project_from_forge
+    @project.enlistments_attributes = { '0' => { 'code_location_attributes' => { 'scm_type' => 'git',
```

Comment:
Although the ticket description vaguely mentions github, this api must support all SCMs. Hence rename `'git'` here to `params[:scm_type] || 'git'`.